Binary files gst-plugins-good_11-11-10_/common/.git/index and gst-plugins-good_11-11-10/common/.git/index differ
diff -ur gst-plugins-good_11-11-10_/ext/soup/gstsouphttpsrc.c gst-plugins-good_11-11-10/ext/soup/gstsouphttpsrc.c
--- gst-plugins-good_11-11-10_/ext/soup/gstsouphttpsrc.c	2011-11-10 21:26:29.000000000 +0400
+++ gst-plugins-good_11-11-10/ext/soup/gstsouphttpsrc.c	2011-11-13 04:29:11.248352938 +0400
@@ -1267,7 +1267,7 @@
   if (src->proxy == NULL) {
     src->session =
         soup_session_async_new_with_options (SOUP_SESSION_ASYNC_CONTEXT,
-        src->context, SOUP_SESSION_USER_AGENT, src->user_agent,
+        /* src->context */ g_main_context_default(), SOUP_SESSION_USER_AGENT, src->user_agent,
         SOUP_SESSION_TIMEOUT, src->timeout,
 #ifdef HAVE_LIBSOUP_GNOME
         SOUP_SESSION_ADD_FEATURE_BY_TYPE, SOUP_TYPE_PROXY_RESOLVER_GNOME,
@@ -1276,7 +1276,7 @@
   } else {
     src->session =
         soup_session_async_new_with_options (SOUP_SESSION_ASYNC_CONTEXT,
-        src->context, SOUP_SESSION_PROXY_URI, src->proxy,
+        /* src->context */ g_main_context_default(), SOUP_SESSION_PROXY_URI, src->proxy,
         SOUP_SESSION_TIMEOUT, src->timeout,
         SOUP_SESSION_USER_AGENT, src->user_agent, NULL);
   }
Binary files gst-plugins-good_11-11-10_/.git/index and gst-plugins-good_11-11-10/.git/index differ
